spec/system/nested_imageable_spec.rb

```diff
@@ -170,4 +170,17 @@
       expect(page).to have_css ".image-fields", count: 1, visible: :all
     end
   end
+
+  context "Show path" do
+    let(:factory) { (factories - [:proposal, :budget, :future_poll_question_option]).sample }
```

Comment:
There's a test in budget investments checking "Index should show investment descriptive image only when is defined" and another one in proposals checking "Index should show proposal descriptive image only when is defined". Would it make sense to remove those tests and change this one accordingly? :thinking: Maybe we could also include budgets, since budgets can have associated images as well since a few years ago.